lib/galaxy/tools/actions/__init__.py

```diff
@@ -949,7 +949,7 @@ def filter_output(tool, output, incoming):
             if not eval(filter.text.strip(), globals(), incoming):
                 return True  # do not create this dataset
         except Exception as e:
-            log.debug('Tool %s output %s: dataset output filter (%s) failed: %s' % (tool.id, output.name, filter.text, e))
+            log.info('Tool %s output %s: dataset output filter (%s) failed: %s' % (tool.id, output.name, filter.text, e))
```

Comment:
Why not use f-string here too? :)
```suggestion
            log.info(f'Tool {tool.id} output {output.name}: dataset output filter ({filter.text}) failed: {e}')
```